cypress/integration/rich-text/__snapshots__/RichTextEditor.Pasting.spec.ts.snap

```diff
@@ -2507,3 +2507,175 @@ exports[`Rich Text Editor > copy from safari (no href in anchors) > recognizes a
   "data": {},
   "nodeType": "document"
 };
+
+exports[`Rich Text Editor > Lists > normalizes table cells correctly #0`] =
```

Comment:
@IBratWurst Looks like you changed the title during development and now we have two snapshots :)